### PR TITLE
Fixed an error that prevented the project to be built.

### PR DESCRIPTION
**Pull request by Sommerbo**
NB: PR #463 supersedes this PR and is much cleaner

"Using UnityEditor" prevents the project to be built into a stand-alone program.
I fixed this with compiler directives and by replacing the command that required UnityEditor.

### CODE REVIEW

**Comment by bjubes** [comment]
Tested this and it works great!

File: Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs [hunk 1]
```diff
@@ -115,11 +117,51 @@ public void DeleteFile()
...
+    /// <returns></returns>
+    private bool IsDirectoryEmpty(string path)
+    {
+        int fileCount = Directory.GetFiles(path).Length;
+        if (fileCount > 0)
+        {
+            return false;
+        }
+        int dirCount = Directory.GetDirectories(path).Length;
+        if (dirCount > 0)
```
Comment (OzieGamma) [inline Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs]: return dirCount <= 0;

**Comment by Sommerbo** [comment]
I am thinking that maybe I should rewrite it.
We could ditch the UnityEditor dependency and remove directory check since currently there are no way to create or navigate folders.

**Comment by vogonistic** [comment]
Closing as duplicate of #463

**Pull request closed without merging**